### ISSUE TITLE
Typo: "uint64_t, uint64_t" ?

### ISSUE THREAD

**Issue by allanmac** [KhronosGroup/GLSL#94]
The same conversion type is repeated.

https://github.com/KhronosGroup/GLSL/blob/7af180fd19879171437d0665b6622d14d0d83164/extensions/ext/GL_EXT_shader_explicit_arithmetic_types.txt#L505

```
 The following table shows allowed integral conversions:
      -------------------------------------------------------------------------
      | Type of    |     Can be implicitly converted to                       |
      | expression |                                                          |
      -------------------------------------------------------------------------
      | ...        |                                                          |
      | uint32_t   | uint64_t, uint64_t                                       |
      | ...        |                                                          |
      -------------------------------------------------------------------------
```

**Comment by abhilash1910** [comment]
hi,@pdaniell-nv,@allanmac . If it's ok ,can I create a PR to update these changes?

**Comment by pdaniell-nv** [comment]
@abhilash1910 yes that would be great, thanks.

**Comment by abhilash1910** [comment]
Hi @pdaniell-nv  could you check out this PR #101.Thanks

**Comment by allanmac** [comment]
> Yes, this should be "int64_t, uint64_t". @allanmac Do you want to create a PR to fix this?

Ooops, I didn't see this.  Thanks @abhilash1910.

**Comment by johnkslang** [comment]
This was fixed by #101.
